contrib/openshift/prometheus-connector-template.yaml

```diff
@@ -0,0 +1,180 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: skydive-prometheus-connector
+objects:
+# Skydive Prometheus Connector (Deployment)
+#
+# Containers included in this pod::
+# (1) Skydive Prometheus Connector - Flow exporter (connects to Skydive Analyzer)
+# (2) Prometheus
```

Comment:
Prometheus is deployed as anoter pod ... please change the remark to indicate that it is not in this pod ... but deployed in a second pod

Reply:
OK